spec/features/user_registration_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe "User registration" do
+  
+  let(:reference){ FactoryGirl.build :user}
+  
+  before {
+    visit root_path
+  }
+  
+  it 'works' do
```

Comment:
try to keep `'works'` at a minimum.

Instead:

`it 'results in a User being created'`